code/modules/admin/topic.dm

```diff
@@ -1712,7 +1713,7 @@
 
 		var/mob/living/carbon/human/H = locateUID(href_list["adminspawncookie"])
 		if(!ishuman(H))
-			to_chat(usr, "This can only be used on instances of type /mob/living/carbon/human")
+			to_chat(usr, "<span class='warning'>This can only be used on instances of type /mob</span>/living/carbon/human")
```

Comment:
Spans incorrectly added here

Reply:
Whoops, I'll be more careful when mass-replacing next time. All done